test/conftest.py

```diff
@@ -600,16 +602,17 @@ def running_service(self, bootslot):
             env=env,
         )
 
-        bus = SessionBus()
+        bus = SessionMessageBus()
 
         # Wait for de.pengutronix.rauc to appear on the bus
         timeout = time.monotonic() + 5.0
         while True:
             time.sleep(0.1)
             try:
-                self.proxy = bus.get("de.pengutronix.rauc", "/")
+                self.proxy = bus.get_proxy("de.pengutronix.rauc", "/")
+                self.proxy.Operation
```

Comment:
What's the purpose of this line? I couldn't find it described anywhere.

Reply:
dasbus seems to communicate with the service only when accessing a property for the first time. So fetch the current Operation and ignore the resut.